### PR TITLE
Update redirect-known-folders.md

### PR DESCRIPTION
**Pull request by joshuc**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: OneDrive/redirect-known-folders.md [hunk 1]
```diff
@@ -118,5 +118,9 @@ The OneDrive Known Folder Move Group Policy objects won't work if you previously
   > We recommend using Windows 10 Fall Creators Update (version 1709 or later) or Windows Server 2019 and the current version of OneDrive to get the benefits from Files On-Demand.
   
   1. [Use Migration Manager](/sharepointmigration/mm-get-started) to copy contents in the network file share location to a user's OneDrive, making sure that all contents go into the existing Documents, Pictures, or Desktop folders.
+
+  > [!NOTE]
+  > If Migration Manager will create the Documents, Pictures, or Desktop folders, ensure that Preserve file share permissions is not selected when performing the migration.
```
Comment (JohanFreelancer9) [inline OneDrive/redirect-known-folders.md:123]:
```suggestion
  > If Migration Manager will create the Documents, Pictures, or Desktop folders, ensure that **Preserve file share permissions** is not selected when performing the migration.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @joshuc, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@adeejoseph PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by adeejoseph — APPROVED** [review]

**Comment by adeejoseph** [comment]
Approved and merged.

**Pull request merged**